native/effektio/src/api/conversation.rs

```diff
@@ -54,55 +55,93 @@ impl Conversation {
 
     pub(crate) fn load_latest_message(&self) {
         let room = self.room.clone();
+        let timeline = room.timeline();
+        let mut stream = timeline.signal().to_stream();
         let me = self.clone();
 
         // FIXME: hold this handler!
         RUNTIME.spawn(async move {
-            let (forward, backward) = room
-                .timeline()
-                .await
-                .context("Failed acquiring timeline streams")
-                .unwrap();
-
-            pin_mut!(backward);
-            // try to find the last message in the past.
-            loop {
-                match backward.next().await {
-                    Some(Ok(ev)) => {
-                        info!("conversation timeline backward");
-                        if let Some(msg) = sync_event_to_message(ev, room.clone()) {
-                            me.set_latest_message(msg);
-                            break;
-                        }
+            while let Some(diff) = stream.next().await {
+                match (diff) {
+                    VecDiff::Replace { values } => {
```

Comment:
shouldn't we handle all of these? Or better even, push through the VecDiff over the FFI to let the actual UI handle with the listing?